### DISCUSSION TITLE
compiling libtbb.so.2 errors

### DISCUSSION THREAD

**Discussion by ghost** [skycocker/chromebrew#9596] [Q&A]
How do I correctly compile libtbb?

**Comment by supechicken** [comment]
@uberhacker @satmandu We should ban this stupid kid from this repo, his meaningless issues flooded my email...

**Reply by uberhacker** [reply]
Blocked and comments hidden!

**Comment by satmandu** [comment]
> How do I correctly compile libtbb?

Patches welcome to fix any issues... Consider it a learning experience!
